sources/packages/backend/apps/api/src/route-controllers/config/config.controller.ts

```diff
@@ -37,6 +37,7 @@ export class ConfigController extends BaseController {
         this.configService.maximumIdleTimeForWarningInstitution,
       maximumIdleTimeForWarningAEST:
         this.configService.maximumIdleTimeForWarningAEST,
+      appEnv: this.configService.appEnv,
```

Comment:
DEVOPS and local docker part is missing in PR. 

Please use this as reference PR - https://github.com/bcgov/SIMS/pull/3626/files